modules/recieve_statustext.py

```diff
@@ -0,0 +1,82 @@
+"""
+This module connects to a drone via MAVLink, collects GPS position data from a communications worker, 
+and generates a KML file after receiving the expected number of positions. The process repeats indefinitely.
+"""
+
+import argparse
+import time
+from pathlib import Path
+from pymavlink import mavutil
+
+from modules.common.data_encoding.message_encoding_decoding import decode_bytes_to_position_global
+from modules.common.data_encoding.metadata_encoding_decoding import decode_metadata
+from modules.common.data_encoding import worker_enum
+from modules.common.kml.kml_conversion import positions_to_kml
```

Comment:
For these imports, can you import modules only, and not specific functions/classes? Just a convention thing. ie `import message_encoding_decoding`, then use `message_encoding_decoding.decode_bytes_to_position_global`.

Reply:
okay